src/test/java/seedu/smartnus/storage/JsonAdaptedQuestionTest.java

```diff
@@ -40,29 +40,29 @@ public void toModelType_validQuestionDetails_returnsQuestion() throws Exception
     @Test
     public void toModelType_invalidName_throwsIllegalValueException() {
         JsonAdaptedQuestion question =
-                new JsonAdaptedQuestion(INVALID_NAME, VALID_IMPORTANCE, VALID_TAGS, VALID_CHOICES);
+                new JsonAdaptedQuestion(INVALID_NAME, VALID_IMPORTANCE, VALID_TAGS, VALID_CHOICES, 0, 0);
```

Comment:
Perhaps we should keep it consistent and have VALID_ATTEMPT and VALID_CORRECT constants